src/main/java/com/salesforce/einsteinbot/sdk/client/model/BotSendMessageRequest.java

```diff
@@ -26,15 +29,32 @@ public class BotSendMessageRequest extends BotRequest {
 
   private List<AnyVariable> variables;
   private AnyRequestMessage message;
+  private Optional<String> tz;
+  private Optional<ResponseOptions> responseOptions;
+  private List<Referrer> referrers;
+  private Optional<RichContentCapability> richContentCapabilities;
 
-  BotSendMessageRequest(Optional<String> requestId, Optional<String> runtimeCRC,
+  protected BotSendMessageRequest(Optional<String> requestId, Optional<String> runtimeCRC,
       RequestEnvelopeInterceptor requestEnvelopeInterceptor,
-      List<AnyVariable> variables, AnyRequestMessage message) {
+      List<AnyVariable> variables,
+      AnyRequestMessage message,
+      Optional<String> tz,
+      Optional<ResponseOptions> responseOptions,
+      List<Referrer> referrers,
+      Optional<RichContentCapability> richContentCapabilities) {
```

Comment:
oh, interesting! We didn't have these capabilities in our public SDK before?

Reply:
Yeah, it's overlooked.